pymeasure/instruments/instrument.py

```diff
@@ -26,7 +26,7 @@
 import time
 
 from .common_base import CommonBase
-from ..adapters import VISAAdapter
+from ..adapters.visa import VISAAdapter
```

Comment:
That line is there, in case we did not load all adapters?

Anyways, as it imports explicitly instead of relying on the init.py, I'm fine with that change.

Reply:
Yup, just a change to import explicitly. Maybe it can futureproof if there are changes to adapters init.py